### ISSUE TITLE
The performance of DNN

### ISSUE THREAD

**Issue by ShengleiH** [rupakvignesh/Singing-Voice-Separation#1]
Hello, thanks for your code sharing.

I am also working on singing voice separation tasks. Unfortunately, my result is not good, I get 5.38dB vocal GNSDR. I trained my model on mir-1k dataset with 175 song clips and I did data augmentation.

I am wondering what's your best result?

Thank you a lot :)

**Comment by rupakvignesh** [comment]
My results are around the same range and I'm currently working on revamping the code a little bit. During training, you can also try to reconstruct vocals while shuffling the background accompaniment.

**Comment by ShengleiH** [comment]
@rupakvignesh Thank you for your rapid reply. Do you mean data augmentation? Actually I have done data augmentation by shifting vocal and then combining it with the original background accompaniment. And the paper I referred to achieves 6.44dB vocal GNSDR, have you ever achieved this results?

**Comment by rupakvignesh** [comment]
In the current implementation, my loss function is different from the original paper's (I was keen to see if estimating just vocals (either directly or with time-freq masks) will be good -- but I was not able to get close to the results on paper. 

Right now, I'm implementing the one suggested in paper and it will be up by the end of this week.

**Comment by ShengleiH** [comment]
@rupakvignesh Thank you ~ :)

**Comment by rupakvignesh** [comment]
Hello @ShengleiH, please check out the new commits and let me know if you have any questions.
